### PR TITLE
support-ethereum-addresses-in-the-qr-scanner

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

File: src/views/shared/qr-scanner/qr-scanner.view.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function QRScanner ({ hideMyCode, onSuccess, onError, onClose }) {
    * @returns {void}
    */
   function handleQRScan (result) {
-    if (result && Addresses.isHermezEthereumAddress(result)) {
+    if ((result && Addresses.isHermezEthereumAddress(result)) || (result && Addresses.isEthereumAddress(result))) {
```
Comment (elias-garcia) [inline src/views/shared/qr-scanner/qr-scanner.view.jsx:35]: I would write the if as: `result && (isHermezEthereumAddress(result) || isEthereumAddress(result))`

**Comment by elias-garcia** [comment]
I think that this is not enough to support Ethereum addresses. When you are redirected to the form, the Ethereum address should be converted to a Hermez address (with the prefix at the beginning) and I think that with this change only it will set the input to just the Ethereum address. It won't display the **** in the middle of the address neither.

In the `transaction-form.view` component, we need to remove the L60 and L61 and call `handleReceiverInputChange` function instead, which is already in the same component. This should make it work as expected I think.

File: src/views/shared/qr-scanner/qr-scanner.view.jsx (hunk 1 above)
Comment (mmilenkovic) [reply src/views/shared/qr-scanner/qr-scanner.view.jsx]: Done.

**Comment by mmilenkovic** [comment]
> I think that this is not enough to support Ethereum addresses. When you are redirected to the form, the Ethereum address should be converted to a Hermez address (with the prefix at the beginning) and I think that with this change only it will set the input to just the Ethereum address. It won't display the **** in the middle of the address neither.
> 
> In the `transaction-form.view` component, we need to remove the L60 and L61 and call `handleReceiverInputChange` function instead, which is already in the same component. This should make it work as expected I think.

Done.

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**